### ISSUE TITLE
JetAppointment calendar synchronization

### ISSUE THREAD

**Issue by alphamale-marketing** [CrocoBlock/suggestions#2543]
It would be very helpful for me when JetAppointment looks into the Google Calendar (or other) if the chosen provider has already an appointment and thereby is **blocked**. (like Calendly)

An example for this is a hairdresser: He gets appointments not only over the website, he gets also _phone calls_ or _visits_ in his shop. With the current configuration of JetAppointment a customer could book an appointment over the website where already another customer booked via telephone. To my mind there are many such cases out there and JetAppointment is not useful for this. I already used Calendly for that but I don't like to operate in two different systems.

Is there any chance to get this feature in the future?

**Comment by alexanderpankart** [comment]
That would certainly help. How is it then when 2 places are free at one time? In that case, would each provider have to have their own calendar?

But in and of itself, integration with existing calendars would certainly be helpful for end customers.

**Comment by alphamale-marketing** [comment]
Yes each provider should have its own (digital) calendar. 
Back to the example of the classic hairdresser: Every employee has its own book (=calendar).

**Comment by chaeu** [comment]
A 2-way synchronization with Google Calendar for every provider would be great. Like @alphamale-marketing said, there are many cases where this would be needed to make JetAppointment a viable option.

**Comment by catherinewiltshire** [comment]
Please, please, add this!

**Comment by alphamale-marketing** [comment]
Is there any progress? That synchronization would extend the usecases of JetAppointment incredibly...!

**Comment by alphamale-marketing** [comment]
Can we make some crowdfunding to push this topic? I would pay 500 $ for this feature. ;-)

**Comment by IFTTD** [comment]
I agree this needs to be added but no response on this so I don't think it will come.

**Comment by mrusteberg** [comment]
@Floydtm , could this be done through an Integromat integration just like it's in place for JetBookings?

**Comment by dennzo** [comment]
I would really need this functionality for a client which is using Google Calendars.
So from my side +1.

**Comment by raghuvirkonanki** [comment]
+1 Please introduce this feature. Amelia and Bookly both have it and it's extremely helpful in bringing Suppliers onboard. Meanwhile, can we not use JetBooking to sync and JetAppointment to book the slot? Does this work?

**Comment by esseffraw** [comment]
I really really need this feature! I build a site for a doctor client, and he use Google Calendar to manage his time. Please add this function, and JetAppointment will be the most powerful appointment plugin in the market!

**Comment by DenoBeno1** [comment]
I don't understand how they didn't implement this from the beginning. This feature is fundamental, but sadly i don't see a lot of innovation in crocoblock and their plugins. They built a basic framework, but most of their functions aren't enough for professional business.

**Comment by Rotg20** [comment]
That's a must, it's such a basic need that it makes Jet-Appointment completely unusable in most cases! 
Also, we need to have a smooth way of letting our authors to run their own calendars from **the front-end**!

**Comment by gguzzman** [comment]
I vote for this feature to work with Calendly and allow calendar synchronization. I think this will be an important leap for Jet appontment. Cheers

**Comment by Guessletsdothings** [comment]
Its not a competitive plugin without this feature. Needs a two-way sync with google calendar to be a viable option for 95% of clients.

**Comment by bikeundski** [comment]
I run a bikeschool with self employed bike guides. They have also another supplier for bookings. Theay are working with google calender. The guides could just manage the bookings of the two agencys in one programm without syncronizing by hand every booking.

It would be a big advantage for me to have this feature.

**Comment by StratoAgency** [comment]
it's been a while and I would still love to have a 2-way sync between Google Calendar & JetAppointments... almost every small business I know is using GC to keep track of appointments and we would be able to sell a lot of custom solutions with JetAppointments if it could do a 2-way sync. 
I would personally test the beta and pay for it just to have it. Crocoblock works like a charm but it definetly needs an upgrade on some plugins...

**Comment by mrusteberg** [comment]
We should set up a bounty for this integration. Obviously ideally
officially supported.

**Comment by Microsmith-1** [comment]
Any mention of this from the CB team and any idea if they are considering it? Ive asked many times and got no reply. ive had to walk away from JetAppointments, We cant keep hanging around waiting for things to happen, im now using Appointedd which has a ton of integration, but id still rather use JetEngine cos ive invested in it.

**Comment by StratoAgency** [comment]
@Crocoblock any chance you could tell us if it is something that you are planning to do?

**Comment by Microsmith-1** [comment]
I do wonder tho what is the point of JetAppointments? JetBookings has a two way calendar sync, but its more geared towards room bookings, you cannot select a single day and time, you have to select a date from - to range, i would use JetBooking if it wasnt for that.

**Comment by StratoAgency** [comment]
@MjHead hey, I know this is not your thread but you reply a lot on betas and stuff! Is it something you would consider?

**Comment by bikeundski** [comment]
Hi crocoblock

 From my side there is still the need of the two way sync to google 
calender.
In my case as a bike school working with self employed bikeguides, who 
are getting there guests from diffrent bikeschools, it is necesery to 
work with a tool (GC) which does not belong to a website from a 
bikeschool and which is common used already from the bikeguides.

This should be just a reminder to keep the topic warm.

Regards

Thomas

Am 2024-03-01 11:19, schrieb StratoAgency:

> it's been a while and I would still love to have a 2-way sync between 
> Google Calendar & JetAppointments... almost every small business I know 
> is using GC to keep track of appointments and we would be able to sell 
> a lot of custom solutions with JetAppointments if it could do a 2-way 
> sync.
> I would personally test the beta and pay for it just to have it. 
> Crocoblock works like a charm but it definetly needs an upgrade on some 
> plugins...
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you commented.Message ID: 
> ***@***.***>



Links:
------
[1] 
https://github.com/Crocoblock/suggestions/issues/2543#issuecomment-1973005565
[2]

**Comment by rtorrespr** [comment]
This feature is a must have, I'm currently needing it for two clients. @Crocoblock Please implements this.

**Comment by PERMI-Design** [comment]
I totally agree! A 2-way synchronization with Google Calendar would take JetAppointment to a whole new level.  Please implement this.

**Comment by alvarindev** [comment]
This will be amazing if you include it. It will kill other solutions in the market

Edit: this is available BTW in jetBooking

**Comment by Crocoblock** [comment]
I am closing this issue because it is a copy

Please keep an eye on [it](https://github.com/Crocoblock/suggestions/issues/3517)
